styles/vendor/_rfs.scss

```diff
@@ -0,0 +1,7 @@
+/**
+ * Responsive Font Sizes (RFS)
+ * ===========================
+ * Expose the responsive font sizing (RFS) API
```

Comment:
Of all the comments in the SCSS, this line is the only one that seems necessary to me.  @ghostdevv what do you think?

Reply:
The only other places I'd say the comments make sense are in:

- Theme & Variables (Since these files contain multiple sections)
- Reset (For the same reason, it contains multiple sections)